multisig/simulations/vip-004/vip-004-sepolia/abi/xvsstore.json

```diff
@@ -0,0 +1,235 @@
+[
```

Comment:
Can we get this abi from the core protocol package?

Reply:
We usually copy the ABI files in the VIP folders because they can change in the different versions where they are originally defined, and we can only define one version in the package.json. I would continue with this approach, to guarantee simulations will run successfully in the future